### ISSUE TITLE
Hover dropdown/sub menu doesn't show in Edit mode

### ISSUE THREAD

**Issue by UniversalDesignz** [ThemeFuse/Brizy#584]
Hi @NataliaBB,

1. The main menu doesn't open up (dropdown/sub menu) on hover in edit mode to style it. 
I'm not sure if you logged in to my back end and have noticed the sub menu are already setup on wp menu section. You will be able to see in live view/site, but not on the Live Edit Mode.
2. The sub/child menu level 2, 3 and so on? If it's there then it should automatically show on the opposite side e.g. if it hits the scrollbar on the left it shouldn't go off the page. 

It would be very helpful to adjust margin, padding, height, width, hover style and effect etc.

Kindest Regards,
Jay Vijay

![menu](https://user-images.githubusercontent.com/15095585/45391461-e740f900-b61a-11e8-90bb-d4eedfe02f43.png)

**Comment by vitog** [comment]
This is true, I would like to have more options withe the menu, hover over changes, or descriptions added to the drop down

**Comment by UniversalDesignz** [comment]
Hi @bagrinsergiu 

I haven't heard back from you guys regards to this
